parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -29,6 +29,12 @@ cis_script=/opt/azure/containers/provision_cis.sh
 wait_for_file 3600 1 $cis_script || exit $ERR_FILE_WATCH_TIMEOUT
 source $cis_script
 
+if [[ $OS == $UBUNTU_OS_NAME ]]; then
```

Comment:
@CecileRobertMichon moved this up here to run on every ubuntu CSE invocation (VHD or full install)